cfg/1.11-json/node.yaml

```diff
@@ -0,0 +1,508 @@
+---
+controls:
+version: 1.11
+id: 2
+text: "Worker Node Security Configuration"
+type: "node"
+groups:
+- id: 2.1
+  text: "Kubelet"
+  checks:
+  - id: 2.1.1
+    text: "Ensure that the --allow-privileged argument is set to false (Scored)"
+    audit: "ps -fC $kubeletbin"
```

Comment:
Should this be checking the config file instead of flags?

Reply:
Well... I thought so at first, but then couldn't find anything related to allow-privileged in the configuration format. According to [the docs](https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/):

> The subset of the Kubelet’s configuration that can be configured via a file is defined by the `KubeletConfiguration` struct [here (v1beta1)](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/config/types.go).

And the struct it points to does not have a member for this option, so I assume this configuration is not part of the "subset" that can be defined in JSON or through dynamic configuration and therefore _has_ to be passed at kubelet start-up as an argument.

Reply:
Huh! I totally assumed you could, but I guess not 😄